target/scripts/build/packages.sh

```diff
@@ -0,0 +1,112 @@
+#! /bin/bash
+
+# -eE :: exit on error (do this in functions as well)
+# -u  :: show (and exit) when using unset variables
+# -o pipefail :: exit on error in pipes
+set -eE -u -o pipefail
+
+# shellcheck source=../helpers/log.sh
+source /usr/local/bin/helpers/log.sh
+
+QUIET='-qq' ; [[ ${LOG_LEVEL} =~ ^trace$ ]] && QUIET='-y'
```

Comment:
Regex removed and simplified:
```suggestion
[[ ${LOG_LEVEL} == trace ]] && QUIET='-y' || QUIET='-qq'
```

Reply:
I would likewise do follow-up here, since the log-level-querying is now on `master` and this can be adjusted a bit. I hope this is okay:)

Good catch though!

Reply:
Done in c6aa12436cbd0fc43f8dfac45d482f1e36689234